solid/lib/Controller/StorageController.php

```diff
@@ -304,13 +306,28 @@ public function handleRequest($userId, $path) {
 
 		$dpop = $this->getDpop();
 
+		$error = false;
 		try {
 			$webId = $dpop->getWebId($request);
 		} catch(\Pdsinterop\Solid\Auth\Exception\Exception $e) {
+			$error = $e;
```

Comment:
Not sure, think this might be more readable/understandable?
```php
		$error = false;

		try {
			$webId = $dpop->getWebId($request);
		} catch(\Pdsinterop\Solid\Auth\Exception\Exception $e) {
			$bearer = $this->getBearer();
			try {
				$webId = $bearer->getWebId($request);
			} catch(\Pdsinterop\Solid\Auth\Exception\Exception $e) {
				$error = $e;
			}
		}

		if ($error instanceof \Pdsinterop\Solid\Auth\Exception\Exception) {
 			$response = $this->resourceServer->getResponse()
				->withStatus(Http::STATUS_CONFLICT, "Invalid token");
		
			return $this->respond($response);
		}